src/app/modules/ix-table/components/ix-table-body/ix-table-body.component.ts

```diff
@@ -79,8 +79,18 @@ export class IxTableBodyComponent<T> implements AfterViewInit {
       && (this.dataProvider?.expandedRow?.[this.detailsRowIdentifier] === row?.[this.detailsRowIdentifier]);
   }
 
-  // TODO: Come up with a proper identifier
-  protected trackByIdentity<X>(item: X): X {
-    return item;
+  protected trackRowByIdentity(item: T): string {
+    return this.getTestAttr(item);
```

Comment:
We need to rename `getTestAttr` and `rowTestId`, because it has different meaning now. Or new fields need to be introduced.